NEW RFC: CODEX-ERASURE-CODING

jimstir opened the pull request:
A new RFC for Codex erasure coding mechanism. Details include how clients should encode their data before presenting it to the Codex network.

emizzle commented on codex/raw/erasure-coding.md:
```diff
@@ -0,0 +1,155 @@
+When to storage request is active erasure coding help ensure the dataset is retrievable from the network.
+This is achieved by the dataset that is chunked is restored in retrieveal by erasure coding.****
+When data blocks are abandoned by storage providers,
+the requester can be assured of data retrievability.
```
This is mostly nonsensical. Please use the [Purpose and Scope section](https://github.com/codex-storage/codex-docs-obsidian/blob/main/10%20Notes/Specs/Component%20Specification%20-%20Erasure%20Coding.md#1-purpose-and-scope) from the original component spec

Cofson commented:
@jimstir 

- Spellchecker and grammar issues, I saw immediately from line 1 a typo and a few others. Please make this a priority for all draft specs.
- No actual encoding/decoding pseudocode - if possible maybe ask the Codex team. 
- Line 57: is this supposed to be JS and not protobuf? Usually we have protobuf here and I believe it was like this in other specs too.
- Leopard codec: could use a link to describe it better, maybe a reference?
- Data repair protocol could use a bit more description and details. Can work with the team together on specific issues.

jimstir commented:
> @jimstir
> 
> * Spellchecker and grammar issues, I saw immediately from line 1 a typo and a few others. Please make this a priority for all draft specs.
> * No actual encoding/decoding pseudocode - if possible maybe ask the Codex team.
> * Line 57: is this supposed to be JS and not protobuf? Usually we have protobuf here and I believe it was like this in other specs too.
> * Leopard codec: could use a link to describe it better, maybe a reference?
> * Data repair protocol could use a bit more description and details. Can work with the team together on specific issues.

Thanks for the feedback.
- Fixed the grammar and spelling issues.
- Changed the js to protobuf for the second code snippet.
- leopard was referenced in the beginning of the spec.

Cofson commented:
> > @jimstir
> > 
> > * Spellchecker and grammar issues, I saw immediately from line 1 a typo and a few others. Please make this a priority for all draft specs.
> > * No actual encoding/decoding pseudocode - if possible maybe ask the Codex team.
> > * Line 57: is this supposed to be JS and not protobuf? Usually we have protobuf here and I believe it was like this in other specs too.
> > * Leopard codec: could use a link to describe it better, maybe a reference?
> > * Data repair protocol could use a bit more description and details. Can work with the team together on specific issues.
> 
> Thanks for the feedback.
> 
> * Fixed the grammar and spelling issues.
> * Changed the js to protobuf for the second code snippet.
> * leopard was referenced in the beginning of the spec.

Approved thx.
